css-fonts-5/Overview.bs

```diff
@@ -95,6 +95,11 @@ Issue(4044):
 <h3 id="font-size-adjust-prop">
 Relative sizing: the 'font-size-adjust' property</h3>
 
+The 'font-size-adjust' property is applied after the 'size-adjust' descriptor.
+
+Note: The apparent effect of applying 'font-size-adjust' after 'size-adjust' is that
+'size-adjust' appears to have no effect.
```

Comment:
Purely a wording nit: to me "The apparent effect of ... appears to have no effect" feels clumsy; maybe it could be "The consequence of ... appears to have no effect"?